Meta/qemu-paths.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+
+# sourcing this file will export SERENITY_QEMU_IMG_BIN and SERENITY_QEMU_BIN
+# in such a way that they will contain the correct paths also under WSL
+
+if command -v wslpath >/dev/null; then
+    # existence of wslpath implies we are running inside WSL on Windows
+    # the Windows installation of QEMU adds the install directory of itself
+    # to the registry, we use powershell to get the install directory
+
+    # Use full path to powershell, just in case it is not on the path already
+    QEMU_INSTALL_DIR=$(/mnt/c/Windows/System32/WindowsPowershell/v1.0/powershell.exe -Command Get-ItemPropertyValue 'HKLM:\SOFTWARE\QEMU' -Name Install_Dir)
+
+    if [ -z "$QEMU_INSTALL_DIR" ]; then
+        if [ "$KVM_SUPPORT" -eq "0" ]; then
+            die "Could not determine where QEMU for Windows is installed. Please make sure QEMU is installed or set SERENITY_IMG_BIN and SERENITY_QEMU_BIN if it is already installed."
```

Comment:
Not sure I understand this - how would setting these prevent the script from entering this branch and dying again? Also, `SERENITY_IMG_BIN` -> `SERENITY_QEMU_IMG_BIN`.

Reply:
Fixed the typo.

Although I am not sure I quite understand what you are asking, but I did realize we are assuming this is being sourced from a script that has `die` implemented in it. I will change this to some `echo`+`exit 1` setup.

I don't know why the `$KVM_SUPPORT` check exists, that just came from the original script (I added a comment to that part of the code), and I don't really know what is actually being used for or whether it could/should be removed here.

The `[ -z "$QEMU_INSTALL_DIR" ]` is only true if it is an empty string in which case the command before it failed and we want to bail because we don't know where QEMU would be located on WSL.

Actually maybe we should also check whether `$QEMU_INSTALL_DIR` is set before even trying to find where it is installed. 🤔 Should the KVM check be just removed from here as KVM could never exist on WSL as far as I know?